src/main/java/de/tum/in/www1/artemis/web/rest/dto/pageablesearch/BasePageableSearchDTO.java

```diff
@@ -0,0 +1,67 @@
+package de.tum.in.www1.artemis.web.rest.dto.pageablesearch;
+
+import de.tum.in.www1.artemis.domain.enumeration.SortingOrder;
+import de.tum.in.www1.artemis.web.rest.dto.SearchResultPageDTO;
+
+// TODO (follow-up, as it changes 40+ files) rename this, move & rename PageableSearchDTO, move UserPageableSearchDTO.
```

Comment:
Let's do this and my requested changes in the follow up PR

Reply:
Sounds good. I'll open the PR once this gets merged to avoid merge conflicts if that's ok with you.

Reply:
Follow-up PR: https://github.com/ls1intum/Artemis/pull/8088